extensions-core/kinesis-indexing-service/src/main/java/org/apache/druid/indexing/kinesis/supervisor/KinesisSupervisorTuningConfig.java

```diff
@@ -212,6 +216,12 @@ public Duration getOffsetFetchPeriod()
     return offsetFetchPeriod;
   }
 
+  @JsonProperty
+  public boolean shouldSkipIgnorableShards()
```

Comment:
```suggestion
  public boolean isSkipIgnorableShards()
```

Since this is a serializable POJO, best to adhere to method naming conventions.